src/ol_infrastructure/infrastructure/mongodb_atlas/Pulumi.infrastructure.mongodb_atlas.mitx.Production.yaml

```diff
@@ -0,0 +1,14 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-production
+encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgHmWivPkd877aOUWMwSqd1KAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMbucetPm1Nyed5WWqAgEQgDt0EbWaCbIAwWRPDhRRjDd1tKLOjVkdy5lR9hrlFpwxXFGkwZ4UWiy2Xxv5M9g2EPqoxGsoadLt9bYVEw==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-mitx-production.odl.mit.edu
+  consul:scheme: https
+  environment:business_unit: residential
+  environment:target_vpc: residential_mitx_vpc
+  mongodb_atlas:disk_autoscale_max_gb: '100'
```

Comment:
No need to specify disk size?

Reply:
Shouldn't this be a larger number for residential?

Reply:
Probably, yes. I'll update that now.